_episodes/13-scheduler.md

```diff
@@ -106,18 +106,29 @@ command (or series of commands) that you want to run on the cluster is called a
 *job*, and the process of using a scheduler to run the job is called *batch job
 submission*.
 
-In this case, the job we want to run is just a shell script. Let's create a
-demo shell script to run as a test. The landing pad will have a number of
-terminal-based text editors installed. Use whichever you prefer. Unsure? `nano`
-is a pretty good, basic choice.
+In this case, the job we want to run is just a shell script -- essentially a
```

Comment:
```suggestion
In this case, the job we want to run is a shell script -- essentially a
```

remove dismissive language around shell scripting, which some learners may have only relatively recently learned about/how to work with.